addons/doris/config/doris-be-config-constraint.cue

```diff
@@ -0,0 +1,486 @@
+#DorisbeParameter: {
+    // Declare a selection policy for servers that have a lot of ip. Note that at most one ip should match this list. This is a list in semicolon-separated format, using CIDR notation, such as 10.10.10.0/24. If no ip address matches this rule, a random IP address is selected. default ''
+    priority_networks: string
+    // set current date for java_opts
+    CUR_DATE: string
+	// log path
+	PPROF_TMPDIR: string
+    // java_opts
+    JAVA_OPTS: string
+    // java_opts_jdk_9
+    JAVA_OPTS_FOR_JDK_9: string
+	// JEMALLOC CONF
+	JEMALLOC_CONF: string
+	// JEMALLOC PROF PRFIX default ""
+	JEMALLOC_PROF_PRFIX: string
+    // system log level
+    sys_log_level: string
+	// Port number of the thrift server on BE, used to receive requests from FE default 9060
+	be_port: int
+	// Service port of the http server on BE default 8040
+	webserver_port: int
+	// The heartbeat service port (thrift) on the BE is used to receive heartbeats from the FE default 9050
+	heartbeat_service_port: int
+	// The port of the brpc on the BE, used for communication between the BE default 9060
+	brpc_port: int
+    // Whether https is supported. If yes, configure ssl_certificate_path and ssl_private_key_path in be.conf default false
+    enable_https: bool
```

Comment:
Maybe format the cue files： `cue fmt addons/doris/configs/*.cue`

Reply:
thanks,I have formatted  the cue files